source/features/arrow-up-add-labels.js

```diff
@@ -0,0 +1,25 @@
+import select from 'select-dom';
+import delegate from 'delegate';
+
+function changeSelection(labels, fromIndex, toIndex) {
+	labels[fromIndex].classList.remove('navigation-focus');
+	labels[fromIndex].setAttribute('aria-selected', 'false');
+	labels[toIndex].classList.add('navigation-focus');
+	labels[toIndex].setAttribute('aria-selected', 'true');
+}
+
+export default function () {
+	delegate('.label-select-menu', '.js-issue-labels-menu-content', 'keydown', event => {
```

Comment:
`delegate` will pile up after page changes because it's inside `ajaxedPagesHandler`. Ensure that it's only set once

Reply:
Yeah, sadly that's the only way I can think of:

```js
global alreadySet = false

if alreadySet
	return

delegate(...)
alreadySet = true
```